src/components/Passports/styles.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Flex, Text, Icon } from "rimble-ui";
+import styled from "styled-components";
+
+export const Container = styled(Flex).attrs(() => ({
+  flexWrap: "wrap",
+  alignItems: "center",
+  justifyContent: "center",
+  mb: 3,
+  pt: 3,
+  pb: 4,
+  borderBottom: "1px solid #DFDFDF",
+  width: "100%"
+}))`
+  transition: transform 0.5s ease-in-out;
+  cursor: pointer;
+`;
+
+export const PassportCover = styled(Flex).attrs(({ shortName }) => ({
+  flexDirection: "column",
+  bg: `passport${shortName}`,
+  m:2,
+  px: 3,
+  py: 2,
+  height: 4,
+  justifyContent: "space-between"
+}))`
+  border-radius: 4px 8px 8px 4px;
+  box-shadow: inset -1px 0 rgba(255, 255, 255, 0.2),
+    0 1px 3px rgba(0, 0, 0, 0.2);
+  position: relative;
+  border: 2px solid #333;
+
+  @supports (background-blend-mode: color-burn) {
+    background-blend-mode: color-burn;
+    background-image: url("https://www.toptal.com/designers/subtlepatterns/patterns/papyrus.png");
```

Comment:
I would avoid hotlinking the background. Can we make it as a local asset by downloading it?

Remember to add the credits, something along those lines would work:

> Background made by Olivia Harmon, https://www.toptal.com/designers/subtlepatterns/papyrus-pattern/

Reply:
Good point :)